main.tf

```diff
@@ -6,9 +6,11 @@ module "final_snapshot_label" {
 }
```

Comment:
It would be good to document this logic in the code with a reference

```
  # PostgreSQL does not use options and option groups. PostgreSQL uses extensions and modules to provide additional features.
  # See https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithOptionGroups.html
```